### PR TITLE
docs: Update documentation and reduce version in preparation for publishing

### PR DESCRIPTION
**Pull request by ggroode**
Updates the documentation in pyproject.toml and Readme.md to prepare for publishing the package to pypi. Also adds example usage to the readthedocs.

### CODE REVIEW

File: docs/getting_started/quickstart.md [hunk 1]
```diff
@@ -20,9 +20,19 @@ export PROMPTGUARD_API_KEY="..."
 PromptGuard offers two main functions: `sanitize()` and `desanitize()`. `sanitize()` takes a string and returns a sanitized (i.e. encrypted and redacted) version of it, while `desanitize()` takes a sanitized string and returns the original string.
 
 ```python
+import promptguard as pg
+
+input_text = "Some string with sensitive PII"
```
Comment (HernanGatta) [inline docs/getting_started/quickstart.md]: Perhaps include actual simulated PII here? That way, if someone runs this as-is, they'll see a redaction in the result.

**Review by HernanGatta — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # PromptGuard
-PromptGuard enables applications to leverage the power of language models without compromising user privacy. This repo contains the PromptGuard Python library, which provides a simple interface for interacting with the PromptGuard API. More information about PromptGuard can be found in the [documentation](https://promptguard.readthedocs.io/).
+PromptGuard enables applications to leverage the power of language models while preserving user privacy. This repo contains the PromptGuard Python library, which provides a simple interface for interacting with the PromptGuard API. More information about PromptGuard can be found in the [documentation](https://promptguard.readthedocs.io/).
+
+**API Stability:** This package is still in development. As such, its API may
```
Comment (octaviansima) [inline README.md:4]: We should avoid overloading the term API -- we use it above to indicate the actual service. What if we call this the Python API, and the PromptGuard API above the PromptGuard Service?

File: docs/getting_started/quickstart.md (hunk 1 above)
Comment (octaviansima) [inline docs/getting_started/quickstart.md]: Nit: for better example usage, let's have this string actually contain sensitive information.

File: python-package-prompt-guard/README.md
Comment (octaviansima) [inline python-package-prompt-guard/README.md:1]: Can we avoid just copying the same README in the repo root?

Comment (ggroode) [reply python-package-prompt-guard/README.md:1]: Not 100% sure, I made the same comment when I first ported it over but not sure what the solution is. The readme shown on pypi is the one inside the python package folder but the one shown on github when you first go to the repo is the one in the repo root. I will take a look and see if you can specify on pypi to use a different readme than the one in the python package folder.

Comment (octaviansima) [reply python-package-prompt-guard/README.md:1]: If that doesn't work you can use a [symlink](https://www.freecodecamp.org/news/linux-ln-how-to-create-a-symbolic-link-in-linux-example-bash-command/)?

Comment (ggroode) [reply python-package-prompt-guard/README.md:1]: Sure I will try.

File: docs/getting_started/quickstart.md [hunk 3]
```diff
@@ -20,9 +20,19 @@ export PROMPTGUARD_API_KEY="..."
...
+sanitized_response = pg.sanitize(text = input_text)
 ```
+The `sanitized_response` contains both the `sanitized_text` as well as `secure_context` which must be passed to the followup call to `desanitize()`.
```
Comment (chester-leung) [inline docs/getting_started/quickstart.md:28]: Please add an example of the `sanitized_response` here, and describe what its contents are

Maybe a new code block with

```
> print sanitized_response
{contents_of_response}
```

Then please go through the contents and describe what they are

File: docs/getting_started/quickstart.md [hunk 4]
```diff
@@ -20,9 +20,19 @@ export PROMPTGUARD_API_KEY="..."
...
+sanitized_response = pg.sanitize(text = input_text)
 ```
+The `sanitized_response` contains both the `sanitized_text` as well as `secure_context` which must be passed to the followup call to `desanitize()`. 
 
 ```python
+# Assume that sanitized_response.sanitized_text was passed into an LLM of your
+# choice, and the final output is saved to 'llm_output"
+desanitized_response = pg.desanitize(sanitized_text = llm_output, secure_context = sanitized_response.secure_context)
+
+print(desanitized_response.desanitized_text)
```
Comment (chester-leung) [inline docs/getting_started/quickstart.md]: Please add an example of what the full desanitized response contains

**Review by zizhong — APPROVED** [review]
Looks good otherwise, except more explanatory examples.

File: python-package-prompt-guard/README.md
Comment (ggroode) [reply python-package-prompt-guard/README.md:1]: Discussed offline, we will keep duplicate Readme's so they can eventually have divergent content.

File: python-package-prompt-guard/src/promptguard/promptguard_service.py [hunk 5]
```diff
@@ -16,7 +16,7 @@
 @dataclass
 class SanitizeResponse:
     sanitized_text: str
-    secure_context: bytes
+    secure_context: str
```
Comment (ggroode) [inline python-package-prompt-guard/src/promptguard/promptguard_service.py:22]: While the go server technically returns a byte object, it is converted to a string by the JSONification before the python code sees it.

**Review by chester-leung — APPROVED** [review]
Thanks for addressing comments.

Please add a description to the PR before merging

File: python-package-prompt-guard/src/promptguard/promptguard_service.py (hunk 5 above)
Comment (octaviansima) [reply python-package-prompt-guard/src/promptguard/promptguard_service.py:22]: Can you add this as a comment?

**Review by octaviansima — APPROVED** [review]

**Pull request merged**